### PR TITLE
Upgrade to ejson2env 2.0.5

### PR DESCRIPTION
**Pull request by mattpatterson94**
## Context

Alongside upgrading ejson2env, this PR adds support for Mac and BSD.

### CODE REVIEW

File: hooks/post-checkout [hunk 1]
```diff
@@ -14,7 +15,26 @@ case "$arch" in
     exit 1
 esac
 
-EJSON2ENV_BIN_PATH="$EJSON2ENV_PLUGIN_ROOT/ejson2env-${arch}"
+os=$OSTYPE
```
Comment (orien) [inline hooks/post-checkout:18]: Where does `OSTYPE` come from? Is it a new and required parameter?

**Review by orien — COMMENTED** [review]
IMO storing binaries in _source_ code control is a bad idea. This the existing state of affairs for this project so I guess it's ok. 🤷

File: hooks/post-checkout (hunk 1 above)
Comment (orien) [reply hooks/post-checkout:18]: Oh, this is provided by each operating system! TIL

```
❯ echo $OSTYPE
darwin21.0
```

**Review by orien — APPROVED** [review]

**Comment by mattpatterson94** [comment]
@orien Looks like some plugins use docker instead. An approach like that does seem like a better idea, but was hoping to get _some_ support happening in the meantime

**Pull request merged**